src/reporter.ts

```diff
@@ -1,6 +1,7 @@
+import assert from "node:assert";
```

Comment:
any reason this was added? we've got another request to [add browser and edge support](https://github.com/lostfictions/znv/issues/2), so if we can avoid bringing in new dependencies from node core that would be ideal.

Reply:
Thank you. I think this was a leftover of a test I did when I upgraded `zod` to prevent a typescript error. However right now it seems to have no effect so I removed it.